skbio/diversity/tests/test_driver.py

```diff
@@ -170,19 +170,19 @@ def test_empty(self):
         # empty vector
         actual = alpha_diversity('sobs',
                                  np.array([], dtype=np.int64))
-        expected = pd.Series([0])
+        expected = pd.Series([0]).astype(int)
```

Comment:
You can do `pd.Series([0], dtype=int)`, which is simpler. (Assuming it works...)